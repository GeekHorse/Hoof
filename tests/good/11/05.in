new right one done
new in suboneb done
out
new in subonea done
save
