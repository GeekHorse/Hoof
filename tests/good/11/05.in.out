new right one done
new in subonea done
new down suboneb done
out
